pybamm/models/submodels/electrode/base_electrode.py

```diff
@@ -196,5 +197,32 @@ def _get_standard_whole_cell_variables(self, variables):
                     phi_s_cn, phi_s_cp
                 )
             )
+            # Contact resistance alters the voltage. To make sure it works for all
+            # operating modes, a new variable "Contact voltage [V]" is created instead
+            # of changing the existing "Terminal voltage [V]".
+            V_dim = variables["Terminal voltage [V]"]
+            if "Current [A]" in variables:
+                I = variables["Current [A]"]
+                V_contact_dim = V_dim - I * param.R_contact
+            elif self.options["operating mode"] == "explicit power":
```

Comment:
In this case, wouldn't P=I*V_contact? And similar for explicit resistance? As in, the measured power would include the contact resistance? Or am I wrong?

It seems to me the cleanest way to do this would be to have a "contact resistance" option (so that the parameter is optional as @tinosulzer suggests) and then not allow "contact resistance" and "explicit power/resistance"?